src/tests/e2e/feature/main/main.cy.ts

```diff
@@ -0,0 +1,38 @@
+export {}
+
+const resultPage = (): void => {
+  const URL = 'http://localhost:3000'
+  cy.visit(URL)
+}
+
+describe('첫 진입 페이지 ', () => {
+  beforeEach(() => {
+    resultPage()
+  })
+  it('배너가 텍스트가 보인다.', () => {
```

Comment:
```suggestion
  it('배너의 텍스트가 보인다.', () => {
```
@price-offer/offer-fe 
test assertion 논의도 금주 위클리 때 같이 논의 되면 좋을 것 같습니다!

Reply:
넵!! 좋습니다!! 
저도 일단 동작여부 확인이 우선이라 TC퀄리티는 챙기지 못하였네요 ㅎㅎㅎ ㅠㅠㅠㅠㅠ